admin/app/com/lucidchart/piezo/admin/controllers/Triggers.scala

```diff
@@ -317,6 +320,42 @@ class Triggers(schedulerFactory: WorkerSchedulerFactory) extends Controller {
     }
   }
 
+  def pauseTrigger(group: String, name: String) = Action { implicit request =>
+    val jobKey = new TriggerKey(name, group)
```

Comment:
Should be 'triggerKey', since you are pausing the trigger, rather than the job. Same thing in the next functon